composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
Don't we need to provide an implementation of PSR-3? Like is done with [DiactorosConfiguration](https://github.com/equip/framework/blob/master/src/Configuration/DiactorosConfiguration.php).